Recolor.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * Class to recolour a transparent image preserving transparency
+ *
+ * @author Robin Andrews (prandrews@hotmail.co.uk)
+ * @author David Yell (neon1024@gmail.com)
+ * @license MIT
+ */
+class Recolor {
+
+    /**
+     * Where the image stored
+     *
+     * @var string
+     */
+    private $imageFolder = __DIR__ . DIRECTORY_SEPARATOR . 'images' . DIRECTORY_SEPARATOR;
+
+    /**
+     * Process an image file
+     *
+     * @return void
+     */
+    public function image($imageName, $newFilename)
```

Comment:
So the `public` here is another visibility keyword, which means that this method can be used by any code which can instantiate an instance of the class.